src/components/ThumbnailImage.js

```diff
@@ -19,26 +19,45 @@ const propTypes = {
     /** Do the urls require an authToken? */
     isAuthTokenRequired: PropTypes.bool.isRequired,
 
+    /** Width of the thumbnail image */
+    imageWidth: PropTypes.number,
+
+    /** Height of the thumbnail image */
+    imageHeight: PropTypes.number,
+
     ...windowDimensionsPropTypes,
 };
 
 const defaultProps = {
     style: {},
+    imageWidth: 200,
+    imageHeight: 200,
 };
 
 class ThumbnailImage extends PureComponent {
     constructor(props) {
         super(props);
 
         this.updateImageSize = this.updateImageSize.bind(this);
-
+        const {thumbnailWidth, thumbnailHeight} = this.calculateThumbnailImageSize(props.imageWidth, props.imageHeight);
         this.state = {
-            thumbnailWidth: 200,
-            thumbnailHeight: 200,
+            thumbnailWidth,
+            thumbnailHeight,
         };
     }
 
-    updateImageSize({width, height}) {
+    /**
+     * Compute the thumbnails width and height given original image dimensions.
+     *
+     * @param {Number} width - Width of the original image.
+     * @param {Number} height - Height of the original image.
+     * @returns {Object} - Object containing thumbnails width and height.
+     */
+    calculateThumbnailImageSize(width, height) {
+        if (!width || !height) {
+            return {};
```

Comment:
@allroundexperts any idea why this was added here. We're thinking to remove in this PR and we believe it will never be undefined. Just wanted to make sure, we don't miss a case and introduce a regression https://github.com/Expensify/App/pull/20892/files